plugins/WireframeView/wireframe.shader

```diff
@@ -0,0 +1,91 @@
+[shaders]
+vertex =
+    uniform highp mat4 u_model_view_projection_matrix;
+
+    attribute highp vec4 a_vertex;
+
+    void main()
+    {
+        gl_Position = u_model_view_projection_matrix * a_vertex;
+    }
+
+fragment =
+    uniform lowp vec4 u_color;
+
+    void main()
+    {
+        gl_FragColor = u_color;
```

Comment:
Yeah without geometry shaders the wireframe view is not possible unless you do computations on the CPU. 

Is there a way to disable the view if the GPU doesn't suport 4.1?